react/features/app/actions.any.ts

```diff
@@ -83,4 +92,51 @@ export function reloadWithStoredParams() {
     };
 }
 
+/**
+ * Checks whether tokenAuthUrl is set, we have a jwt token that will expire soon
+ * and redirect to the auth url to obtain new token if this is the case.
+ *
+ * @param {Dispatch} dispatch - The Redux dispatch function.
+ * @param {Function} getState - The Redux state.
+ * @param {Function} failureCallback - The callback on failure to obtain auth url.
```

Comment:
This looks weird, isn't there a success one?

Reply:
Nope ... this is all because of the Promise for `window.crypto.subtle.digest` we recently added ... so it will fail if that call fails ...